xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/liveDataSource.js

```diff
@@ -93,6 +104,7 @@ define('xwiki-livedata-source', ['module', 'jquery'], function(module, $) {
   };
 
   var toLiveData = function(data) {
+    entriesRequest = null;
```

Comment:
I'm not sure to understand why a reset to null is needed here. It might deserve a comment.